com.unity.sg2/Editor/GraphUI/GraphElements/ModelUI/Inspector/TargetListPropertyField.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.GraphToolsFoundation.CommandStateObserver;
+using UnityEngine.UIElements;
+
+namespace UnityEditor.ShaderGraph.GraphUI
+{
+    class TargetSettingsListPropertyField<T> : ListPropertyField
```

Comment:
I mostly left this alone to avoid breakage, only removing/changing what was covered by the new superclass. Let me know if there's anything else that looks like it should be changed.